common-pg/src/main/scala/com/socrata/pg/store/RollupManager.scala

```diff
@@ -192,25 +192,36 @@ class RollupManager(pgu: PGSecondaryUniverse[SoQLType, SoQLValue], copyInfo: Cop
 
             // OK COOL!  We're all ready to go!
 
-            val create = d"CREATE TABLE" +#+ Doc(rollupInfo.tableName) +#+ d"(" ++
+            val tableName = Doc(rollupInfo.tableName)
+            val create = d"CREATE TABLE" +#+ tableName +#+ d"(" ++
               schema.iterator.flatMap { case (label, rep) =>
                 rep.expandedDatabaseColumns(label).zip(rep.expandedDatabaseTypes).map { case (colName, colTyp) =>
                   colName +#+ colTyp
                 }
               }.toSeq.punctuate(d",").hsep ++
               d")" ++ Doc(tablespaceSql)
 
-            val populate = d"INSERT INTO" +#+ Doc(rollupInfo.tableName) +#+ d"(" ++
+            val populate = d"INSERT INTO" +#+ tableName +#+ d"(" ++
               schema.iterator.flatMap { case (label, rep) =>
                 rep.expandedDatabaseColumns(label)
               }.toSeq.punctuate(d",").hsep ++
               d")" +#+
               sql
 
-            val searchIdxSql = new SoQLRewriteSearch[DatabaseNamesMetaTypes](searchBeforeQuery = true).
-              searchTerm(schema).
-              map { searchTerm =>
-                d"CREATE INDEX IF NOT EXISTS idx_" ++ Doc(rollupInfo.tableName) ++ d"_fts ON" +#+ Doc(rollupInfo.tableName) +#+ d"USING GIN (to_tsvector('english'," ++ searchTerm ++ d"))" ++ Doc(tablespaceSql)
+            val searchIdxSql =
+              foundTables.initialQuery match {
+                case FoundTables.Saved(_) =>
```

Comment:
Can you explain why `FoundTables.Saved(_)` is indicative of a materialized view?

Reply:
Because that means "this was created as 'the same query as a saved view'" - i.e., we're materializing the output of that view.